CODEOWNERS

```diff
@@ -195,6 +195,9 @@
 # - @cilium/wireguard:
 #   Maintain the kernel WireGuard configuration and datapath impacts related to
 #   ensuring traffic is encrypted correctly when WireGuard mode is enabled.
+# - @cilium/ipsec:
```

Comment:
nit, could you put this in alphabetic order wrt the other codeowners?

Reply:
:facepalm: I added it here to take WireGuard as an example and then forgot to move the entry...